examples/storage/s3/README.md

```diff
@@ -0,0 +1,176 @@
+# Using Incremental Learning Job in Helmet Detection Scenario on S3
+
+This example based on the example: [Using Incremental Learning Job in Helmet Detection Scenario](/examples/incremental_learning/helmet_detection/README.md)
+

```

Comment:
since we add a new field `credentialName` for shared storage in release 0.2.0, we need to say something about upgrading the crd definition

Reply:
ok